docs/getting-started/index.md

```diff
@@ -45,6 +45,7 @@ Do you want to use a framework? Native integrations with the most popular librar
 * Integrate CKEditor&nbsp;5 with React using  {@link getting-started/integrations-cdn/react-default-cdn CDN} or {@link getting-started/integrations/react-default-npm npm/ZIP}
 * Integrate CKEditor&nbsp;5 with Next.js using {@link getting-started/integrations-cdn/next-js CDN} or {@link getting-started/integrations/next-js npm/ZIP}
 * Integrate CKEditor&nbsp;5 with Vue.js 3.x using {@link getting-started/integrations-cdn/vuejs-v3 CDN} or {@link getting-started/integrations/vuejs-v3 npm/ZIP}
+* Integrate CKEditor&nbsp;5 with Vue.js 2.x using {@link getting-started/integrations/vuejs-v2 npm/ZIP}
```

Comment:
No love for CDN in here?

Reply:
Tbh, I wouldn't promote it. The integration differs from other JS integrations.

Reply:
Sure.